config/locales/client.en.yml

```diff
@@ -1448,7 +1448,7 @@ en:
         redeemed_tab: "Redeemed"
         redeemed_tab_with_count: "Redeemed (%{count})"
         invited_via: "Invitation"
-        invited_via_link: "link (%{count} / %{max} redeemed)"
+        invited_via_link: "link %{key} (%{count} / %{max} redeemed)"
```

Comment:
I wonder if we've approached the point of where it makes sense to split these on the UI into separate HTML elements.

Reply:
This is something I asked myself too, but I kept this as I do not like the idea of many tiny elements just to display this text.